lib/manageiq/appliance_console/certificate_authority.rb

```diff
@@ -92,6 +98,9 @@ def configure_pgserver
         # no need for username/password since not writing database.yml
         InternalDatabaseConfiguration.new(:ssl => true).configure_postgres
         LinuxAdmin::Service.new(PostgresAdmin.service_name).restart
+
+        say "configuring certmonger to start on reboot"
+        LinuxAdmin::Service.new("certmonger").enable.start
```

Comment:
Should this maybe be a class method on the `Certificate` class?

Either way, I think making it a method somewhere would be better than this same call being in three places.